docs/reference/Image.rst

```diff
@@ -330,6 +330,34 @@ Instances of the :py:class:`Image` class have the following attributes:
 
     Unless noted elsewhere, this dictionary does not affect saving files.
 
+.. py:attribute:: Image.is_animated
+    :type: bool
+
+    ``True`` if this image has more than one frame, or ``False`` otherwise.
+
+    This attribute is only defined by image plugins that support animated images.
+    Plugins may leave this attribute undefined if they don't support loading
+    animated images, even if the given format supports animated images.
+
+    To check whether an image is animated regardless of its format, use
+    ``getattr(image, "is_animated", False)``.
```

Comment:
The problem of https://github.com/python-pillow/Pillow/pull/4628#issuecomment-650542773 still applies: this will return `False` for some animated images (and below return 1 for `n_frames`).